jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/GltfUtils.java

```diff
@@ -425,12 +425,21 @@ private static void populateByteArray(byte[] array, LittleEndien stream, int cou
         int stride = Math.max(dataLength, byteStride);
         int end = count * stride + byteOffset;
         stream.skipBytes(byteOffset);
+
+        if (dataLength == stride) {
+            byte[] buffer = new byte[end - index];
+            stream.read(buffer, 0, buffer.length);
```

Comment:
Note: read() may read fewer bytes than buffer.length and it's bad practice to ignore its return value.  I'd suggest using readFully() instead but JME's readFully() is broken in a similar way.  This is a really dangerous way to read because failures will look really strange and not necessarily point to this read() code.